src/griffe/finder.py

```diff
@@ -338,22 +343,21 @@ def _handle_editable_module(path: Path):  # noqa: WPS231
         editable_lines = path.read_text(encoding="utf8").splitlines(keepends=False)
     except FileNotFoundError:
         raise UnhandledEditableModuleError(path)
+
```

Comment:
Please remove this style changed :smile: 
We should either space the code in the whole function, or not at all :stuck_out_tongue:

Reply:
I went ahead and committed a suggestion directly from GitHub :slightly_smiling_face: